### PR TITLE
add analytics api and collector

### PR DESCRIPTION
**Pull request by k-ayache**
add analytics api and collector docker images

### CODE REVIEW

**Comment by k-ayache** [comment]
add partitions argument in configuration of analytics-api, it is required. and correct the syntaxe of  discovery arguments in analytics-api configuration

**Comment by k-ayache** [comment]
my branch is updated with master origin and merged with my modifications

File: default.nix [hunk 1]
```diff
@@ -49,6 +49,14 @@ in rec {
       name = "opencontrail/control";
       command = "${contrail32Cw.control}/bin/contrail-control --conf_file ${configuration.control}";
     }
+    { attr = "contrailCollector";
+      name = "opencontrail/collector";
+      command = "${contrail32Cw.collector}/bin/contrail-collector --conf_file ${configuration.collector}";
+    }
+    { attr = "contrailAnalyticsApi";
+      name = "opencontrail/analytics/api";
```
Comment (nlewo) [inline default.nix]: Can you rename it to opencontrail/anaylitics-api ?
The format is namespace/imageName.

File: configuration.nix [hunk 2]
```diff
@@ -77,4 +79,74 @@ control = pkgs.writeTextFile {
...
+    [API_SERVER]
+    api_server_list = 10.0.0.3:8082
+    '';
+   
+  };
+
+analytics-api = pkgs.writeTextFile {
+  name = "contrail-analytics-api.conf";
+  text = ''
+
```
Comment (nlewo) [inline configuration.nix]: Not needed

**Pull request merged**